livestyled/schemas/device_preference.py

```diff
@@ -0,0 +1,17 @@
+from marshmallow import EXCLUDE, fields, Schema
+
+from livestyled.models.device_preference import DevicePreference
+
+
+class DevicePreferenceSchema(Schema):
+    class Meta:
+        unknown = EXCLUDE
+        model = DevicePreference
+        api_type = 'device_preferences'
+        url = 'v4/device_preferences'
+
+    id = fields.Int()
+    created_at = fields.String()
```

Comment:
similar here, need a data_key to ensure the key format matches what is in the API response (camelCase)